.github/workflows/tests.yml

```diff
@@ -36,11 +36,9 @@ jobs:
           ${{ runner.os }}-php-
     - name: Install dependencies
       if: steps.composer-cache.outputs.cache-hit != 'true'
-      uses: php-actions/composer@v4
+      uses: MilesChou/composer-action/5.5/install@master
```

Comment:
@tristan-orourke Shouldn't this be using PHP version 7.2 instead pf 5.5?

`uses: MilesChou/composer-action/7.2/install@master`

Reply:
Oops! Copy paste errors! Thanks.